data/de/conferences.yml

```diff
@@ -3,6 +3,7 @@ items:
     name: JakartaOne Livestream German
     location: Location, Location
     date: June 30, 2022
+    start_date: 2022-07-30T00:00:00-00:00
```

Comment:
Every entry of `start_date: 2022-07-30T00:00:00-00:00` is incorrect as it should be 2022-06-30

Reply:
Fixed!